lib/multicell_location/services/here_integration.c

```diff
@@ -45,12 +46,10 @@ BUILD_ASSERT(sizeof(API_APP_ID) > 1, "App ID must be configured");
 	"&app_id="CONFIG_MULTICELL_LOCATION_HERE_APP_ID
 #endif
 
-#define HTTP_REQUEST_HEADER						\
-	"POST "PATH"?"AUTHENTICATION" HTTP/1.1\r\n"			\
-	"Host: "HOSTNAME"\r\n"					        \
-	"Content-Type: application/json\r\n"				\
-	"Connection: close\r\n"						\
-	"Content-Length: %d\r\n\r\n"
+#define REQUEST_URL		API_LOCATE_PATH"?"AUTHENTICATION
+
+#define HEADER_CONTENT_TYPE    "Content-Type: application/json\r\n"
+#define HEADER_CONNECTION      "Connection: close\r\n"
```

Comment:
The REST client context has parameter `keep_alive` which controls whether the socket is closed after the request or not. Could we move the `Connection` header handling to the REST client library as well?

I.e. if `keep_alive` is `true`, include a "Connection: keep-alive" header and if `keep_alive` is `false`, include a "Connection: close" header.

Reply:
I'm fine with keeping this as it is for now. But I guess that even with short(ish) timeouts, it would be possible to perform multiple requests back to back over the same connection.